lightning/src/onion_message/packet.rs

```diff
@@ -314,6 +314,16 @@ pub(crate) enum ControlTlvs {
 	Receive(ReceiveTlvs),
 }
 
+/// A wrapper struct that stores the largest packet length in the given [`BlindedPath`].
+/// This helps us calculate the appropriate padding size for the [`ControlTlvs`] at the time
+/// of writing them.
+pub(crate) struct LengthTlvs {
```

Comment:
How about `ControlTlvsWithPadding`? This gives a better indication of how it is written, which is the purpose of the struct.

Reply:
Build upon the following suggestion to use a generic struct for padding!
Thanks for the suggestion!